gvsbuild/patches/libtiff-4/0001-cmake-remove-.d-postfix.patch

```diff
@@ -14,13 +14,13 @@ index 8cfe7ae3..7a8b423a 100644
 --- a/cmake/WindowsSupport.cmake
 +++ b/cmake/WindowsSupport.cmake
 @@ -25,7 +25,6 @@
- 
- # Debug postfix
+
  if(MSVC)
+     # Debug postfix
 -    set(CMAKE_DEBUG_POSTFIX "d")
```

Comment:
this looks weird... why are we dropping `set(CMAKE_DEBUG_POSTFIX "d")` ?

If this is intentional, then at least we need to change the patch name

Reply:
I just rebased the patch. I'm not sure what this setting does. Does it append a d to target files if debug is set?

Reply:
the patch is needed, otherwise you endup with a library called tiffd.lib when building the debug version

Reply:
Oooh... I see, since this is a patch I was misreading the line as removed, while the line stays but it has a `-` in the patch itself